src/equations/compressible_euler_2d.jl

```diff
@@ -440,6 +440,66 @@ function initial_condition_eoc_test_coupled_euler_gravity(x, t, equations::Compr
   return prim2cons(SVector(rho, v1, v2, p), equations)
 end
 
+"""
+    initial_condition_rti(coordinates, t, equations::CompressibleEulerEquations2D)
```

Comment:
Hm, but this is inconsistent with `initial_condition_khi`. If we change this name, should we change `khi` too?

Reply:
Good point. This might be a breaking change, but now's the time... I would prefer `initial_condition_kelvin_helmholtz_instability` - do you agree @ranocha?

Reply:
Yes, we should rename that one, too, when moving it to the elixir.